features/unattached_commands.feature

```diff
@@ -196,6 +196,23 @@ Feature: Command behaviour when unattached
            | groovy   | no           |
            | hirsute  | no           |
 
+
+    @series.focal
+    Scenario Outline: Useful SSL failure message when there aren't any ca-certs
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I run `apt remove ca-certificates -y` with sudo
+        When I verify that running `ua fix CVE-1800-123456` `as non-root` exits `1`
+        Then I will see the following on stderr:
+            """
+            Cannot verify certificate of authentication server
+            Try installing `ca-certificates` if it isn't already and try again.
+            Failed to access URL: https://ubuntu.com/security/cves/CVE-1800-123456.json
```

Comment:
Here is an example of the new message we'll print if there is an SSL verification error. I just realized I used backticks out of habit and not sure if thats what we want here. Also, should we add another line along the lines of "If that doesn't work, check your openssl configuration." - What do you think? @mitechie @lucasmoura @blackboxsw

Reply:
I think dropping the backticks and adding the extra line is fine.
But let's hear from @mitechie and @blackboxsw on this as well

Reply:
A few comments. I agree that backticks should be quotes instead. 

I'd prefer if the error came first "Failed to access URL:..." and then the other material is the details about the said error. 

Can we tell if the ca-certificates package is installed or not vs leaving it hanging with ambiguity?

Reply:
Ack: will change to quotes and will put "Failed to access..." first.

We could tell if ca-certificates is installed or not - I just wasn't sure how far down the path of debugging the problem we think uaclient should go (e.g. if ca-certificates is installed, do we check if the openssl config file is valid? do we check to see if the cert we got from contracts.canonical.com is what we expected?).

I think checking if ca-certificates is installed or not is a reasonable step for uaclient to take, and we can print a different message accordingly ("Please install ca-certificates" vs "Check your openssl configuration"). We probably shouldn't go any further though. What do you think @mitechie @blackboxsw @lucasmoura ?

Reply:
If it's easy to check the package specifically we know we hit that case for a user and so it seems like a reasonable step to get the user as close to the problem as possible. Thanks for the updates.